tests/unit/drivers/image_generation/test_azure_openai_image_generation_driver.py

```diff
@@ -19,15 +19,15 @@ def test_init(self, driver):
 
     def test_init_requires_endpoint(self):
         with pytest.raises(TypeError):
-            AzureOpenAiDalleImageGenerationDriver(
+            AzureOpenAiImageGenerationDriver(
                 model="dall-e-3", client=Mock(), azure_deployment="dalle-deployment", image_size="512x512"
-            )
+            )  # pyright: ignore
```

Comment:
Ignoring here and on line 30 because we're explicitly testing the response to the missing field.